packages/block-library/src/navigation-link/index.php

```diff
@@ -344,3 +344,26 @@ function register_block_core_navigation_link() {
 	);
 }
 add_action( 'init', 'register_block_core_navigation_link' );
+
+/**
+ * Disables the display of tabs for the Navigation Link block.
```

Comment:
Can we add some context as to why this decision has been taken?

Let's add a link to the discussions that landed us here. I would make a suggestion but I've lost track of where...